src/main/java/omg/group/priuttelegrambot/dto/flags/FlagsDto.java

```diff
@@ -0,0 +1,20 @@
+package omg.group.priuttelegrambot.dto.flags;
+
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Data
+public abstract class FlagsDto {
+    private Long id;
+    private Boolean isWaitingForPhoto;
+    private Boolean isWaitingForRation;
+    private Boolean isWaitingForFeeling;
+    private Boolean isWaitingForChanges;
+    private Boolean isWaitingForContacts;
+    private LocalDate date;
+    private Boolean isChatting;
```

Comment:
```suggestion
    private Boolean chatting;
```
на is называем методы, которые возвращают boolean